libvast/src/format/test.cpp

```diff
@@ -234,6 +234,8 @@ auto default_schema() {
   return result;
 }
 
+using default_randomizer = randomizer<std::mt19937_64>;
```

Comment:
I factored this only to reduce code duplication. The engine makes no difference to us as far as I can see (I would simply put `std::default_random_engine` in there, to be honest), but I changed it nonetheless since you seem to have some preferences.

Reply:
Seems like changing the random engine involves a few more changes than expected. How about we post-pone this and revert HEAD by 1 commit?

Reply:
👍